testcases.py

```diff
@@ -266,6 +266,38 @@ def check(self):
         return True
 
 
+class TestCaseLongRTT(TestCaseHandshake):
+    @staticmethod
+    def abbreviation():
+        return "Y"
+
+    @staticmethod
+    def name():
+        return "longrtt"
+
+    @staticmethod
+    def testname(p: Perspective):
+        return "handshake"
+
+    @staticmethod
+    def scenario() -> str:
+        """ Scenario for the ns3 simulator """
+        return "simple-p2p --delay=750ms --bandwidth=10Mbps --queue=25"
+
+    def check(self):
+        if not super(TestCaseLongRTT, self).check():
+            return False
```

Comment:
Why's this necessary?

Reply:
This calls the `check` method of the parent class (`TestCaseHandshake`), which checks that
* there's only a single handshake
* the transferred files match

Reply:
I should've deleted this comment -- this was really a "why do we need to override this at all" question.